drivers/overlay/ov_network.go

```diff
@@ -298,6 +298,14 @@ func (n *network) initSubnetSandbox(s *subnet) error {
 	return nil
 }
 
+func (n *network) cleanupStaleSandboxes() {
+	for i := 1; i < n.initEpoch; i++ {
```

Comment:
Oh I see, this function is being called in `initSandbox` after the `n.initEpoch` has been incremented...